sampleScripts/Sandia-SEMS/test_tpetra_sandia

```diff
@@ -277,13 +277,19 @@ elif [ "$MACHINE" = "white" ]; then
   #KDD 9/18  OpenMPI3 has problems with Tpetra_ASSUME_CUDA_AWARE_MPI=ON
   #KDD 9/18  module load devpack/20180521/openmpi/3.1.0/gcc/7.2.0/cuda/9.2.88
   #KDD 9/18  This configuration seems to work
-  module load git/2.10.1
-  module load openmpi/2.1.2/gcc/7.2.0/cuda/9.2.88
-  module load cmake/3.9.6
-  module load openblas/0.2.20/gcc/7.2.0
-  module load boost/1.65.1/gcc/7.2.0
-  module load cuda/9.2.88
-  module load netcdf-exo/4.4.1.1/openmpi/2.1.2/gcc/7.2.0/cuda/9.0.176
+  #module load git/2.10.1
```

Comment:
We might want to separate these SEMS script changes into their own pull request.